RSpec/spec/source_spec.rb

```diff
@@ -0,0 +1,19 @@
+require './classes/attributes/source'
+require './classes/item'
+
+describe Source do
+  before :each do
+    @item = Item.new(Time.now.to_i, '2000-10-10', true)
+    @source = Source.new('Shop')
+    @item.source = @source
+  end
+
+  describe '#test methods of class source' do
+    it 'returns #source name matches "Shop"' do
+      expect(@item.source.name).to(eq('Shop'))
+    end
+    it 'returns #@items[0] matches "@items"' do
+      expect(@source.items[0]).to(eq(@item))
+    end
+  end
+end
```

Comment:
- [ ] All in all, we are missing the tests for an `item_spec`

- move_to_archive
- add_genre
- add_author
- add_source
- add_label

Reply:
:heavy_check_mark: Done!